SP-000 add how to release to the README (#34)
Staffbase/github-action-issue-id-collection: README.md

- `tIDs`: List of Ticket-IDs, seperated by comma
- `tagName`: Name of the current Tag

## How to release

> Just edit the DRAFT release in the GitHub UI and publish it with a new git tag.

Reviewer: ```suggestion
Just edit the DRAFT release on the [Releases page](https://github.com/Staffbase/github-action-issue-id-collection/releases)  and publish it with a new git tag.
```
